### PR TITLE
Use response_mode=jwt in client_secret example

### PR DESCRIPTION
**Pull request by yovasx2**
We introduce the use of response_mode=jwt in client_secret_basic example and the JWS verification in both examples
https://identity-online.atlassian.net/browse/DEV-1174

### CODE REVIEW

File: client-secret-basic-example/app.js [hunk 1]
```diff
@@ -30,10 +34,11 @@ app.get('/button/oauth', async (req, res) => {
       nonce,
       scope,
       client_id: clientId,
-      client_secret: clientSecret,
       identity_provider_id: idpId,
       prompt: 'consent',
       response_type: "code",
+      // If FAPI is enabled, this is necessary to prevent InvalidRequest exception
```
Comment (francisco-idpartner) [inline client-secret-basic-example/app.js:40]: Can they disable fapi? I don't think so. I think this should always be set and the comment should be removed

File: client-secret-basic-example/app.js [hunk 2]
```diff
@@ -46,7 +51,17 @@ app.get('/button/oauth', async (req, res) => {
 
 // Callback endpoint called after the user completes the authorization process
 app.get('/button/oauth/callback', async (req, res) => {
-  const { code } = req.query;
+  let code
+  // We need to used the ID provider's issuer URL
+  const issFromCookie = req.cookies.iss;
+  if (req.query.response) {
+    // If FAPI is enabled, the response is a JWS and we need to verify it is emited by the correct issuer
```
Comment (francisco-idpartner) [inline client-secret-basic-example/app.js:58]: FAPI is always enabled, isn't it?

**Review by francisco-idpartner — COMMENTED** [review]
Left a few questions, the code looks good.

As per out last change in process management, please create a JIRA and link the PR to it.

File: client-secret-basic-example/app.js (hunk 1 above)
Comment (yovasx2) [reply client-secret-basic-example/app.js:40]: yeap i will remove it

File: client-secret-basic-example/app.js (hunk 2 above)
Comment (yovasx2) [reply client-secret-basic-example/app.js:58]: yeah that is the idea

**Comment by yovasx2** [comment]
in favor of https://github.com/idpartner-app/examples/pull/4

**Pull request closed without merging**